README.rst

```diff
@@ -56,6 +56,11 @@ Please report feature requests and problems to:
 
 https://github.com/WeblateOrg/weblate/issues
 
+
+Live chat
+----
+Live chat about Weblate is available at Libera.Chat IRC network. The channel name is #weblate . This can be accessed by, for example, http://web.libera.chat or an IRC client installed on your computer.
```

Comment:
```suggestion
Live chat about Weblate is available at Libera.Chat IRC network. The channel name is ``#weblate``. This can be accessed by, for example, https://web.libera.chat/#weblate or an IRC client installed on your computer.
```

- quote the channel name
- use https for web chat
- adjust web chat link to prefill channel